documentation/api/query/v4/operators.markdown

```diff
@@ -203,6 +203,58 @@ When querying a particular endpoint, such as `/pdb/query/v4/nodes`, the endpoint
 the context for the query. Querying the [root] endpoint requires specifying a
 context explicitly.
 
+## Paging operators (`limit`, `offset`, `order_by`)
+
+PuppetDB allows specification of paging clauses within a "from" clause in a
+query or subquery. The `limit` and `offset` operators both accept an
```

Comment:
I guess this is in the "operators" doc, so that's just what we call them. Not sure how I feel about that, but whatever.